test/ETO/ETOCommitment.js

```diff
@@ -616,6 +662,19 @@ contract("ETOCommitment", ([deployer, admin, company, nominee, ...investors]) =>
       ).to.be.rejectedWith("NF_ETO_MAX_TOK_CAP");
     });
 
+    it("reverts on crossing max cap (max investment amount)", async () => {
+      const maxInvestAmount = Q18.mul(5000000);
+      await deployEtoWithTicket({
+        ovrETOTerms: { MAX_TICKET_EUR_ULPS: maxInvestAmount.mul(2) },
+        ovrETOTermsConstraints: { MAX_INVESTMENT_AMOUNT_EUR_ULPS: maxInvestAmount }, // max invest is 5mio
+        ovrTokenTerms: { MAX_NUMBER_OF_TOKENS: Q18 }, // we allow many tokens, so there is no max cap triggered there
+      });
+      await skipTimeTo(publicStartDate.add(1));
+      await expect(investAmount(investors[4], maxInvestAmount.add(Q18), "EUR")).to.be.rejectedWith(
```

Comment:
please use add(1), just one wei should be enough

Reply:
ok, done